src/rnp/fficli.cpp

```diff
@@ -649,6 +651,15 @@ cli_rnp_t::init(const rnp_cfg &cfg)
         return false;
     }
 
+    if (cfg_.has(CFG_ALLOW_SHA1)) {
+        rnp_add_security_rule(ffi,
+                              RNP_FEATURE_HASH_ALG,
+                              "SHA1",
+                              RNP_SECURITY_OVERRIDE,
```

Comment:
RNP_SECURITY_VERIFY_KEY should be used here in this particular case, please see tests for the example.